app/views/admins/orders/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<p id="notice"><%= notice %></p>
```

Comment:
Maybe we can move the notice to `application.html.erb` layout

Reply:
done :)